<launch>
  <node name="visual_servo_with_rotation" pkg="rosovbc" type="visual_servo_with_rotation"/>
  <param name="cameraTopic" type="string" value="/camera/image_raw"/> 
  <param name="robotTopic" type="string" value="/cmd_vel"/>
  

  <!-- with MEtimator -->
<param name="error_threshold" type="double" value="1e-3"/>


  <param name="lambda_first_rot" type="double" value="3"/>
  <param name="mu_first_rot" type="double" value="0.01"/>


  <param name="lambda_combined_rot_trans" type="double" value="5"/>
  <param name="mu_first_rot" type="double" value="0.01"/>


  <param name="alpha" type="double" value="3.0"/>
  <param name="beta" type="double" value="1.0"/>
  <param name="gamma" type="double" value="0.5"/>


<!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="joy_node" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.05" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="0.11" type="double"/>
  <param name="scale_angular" value="0.11" type="double"/>

  <param name="logs" type="string" value="/home/mis-pr/Developpement/catkin_ws/src/rosovbc/logs/"/>
  <param name="data" type="string" value="/home/mis-pr/Developpement/catkin_ws/src/rosovbc/data/"/>
</launch>


